Assets/Scripts/Photos/PhotoAlbum.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+class PhotoAlbum : MonoBehaviour
+{
+    public RawImage albumPhoto, albumPhoto1, albumPhoto2, albumPhoto3;
```

Comment:
I would put this in a list to make it easier to design instead, esp. if we want to change album size.

Reply:
Done!